src/ResourceManager/Automation/Commands.Automation/Cmdlet/ImportAzureAutomationDscNodeConfiguration.cs

```diff
@@ -67,6 +77,7 @@ public override void ExecuteCmdlet()
                     this.AutomationAccountName,
                     this.Path,
                     this.ConfigurationName,
+                    this.IncrementNodeConfigurationBuild.IsPresent,
```

Comment:
If this is parameter is passed, there won't be any prompt and we'll create a new version of NodeConfig in the service. If you do not pass -Force and -IncrementNodeConfigurationBuild, the prompt will show up (If the node configuration exists in the service). If you pass the -Force, it'll overwrite the existing NodeConfiguration .